### PR TITLE
Tech debt - Cleanup some apps and urls

### PR DESCRIPTION
**Pull request by mbeacom**
##### Description

The goal of this PR is to cleanup some of the backend modules by splitting them up, standardizing URLs, etc.

##### Checklist

- [x] Read and conforms to the [contributor guidelines](https://docs.gitcoin.co/mk_contributors/)
- [x] Read and conforms to the [style guidelines](https://docs.gitcoin.co/mk_styleguide/)
- [x] Linter status: 100% pass
- [x] Added relevant tests
- [x] Tests pass and code coverage has not decreased
- [x] Changes don't break existing behavior
- [x] Commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
- [x] [Code documentation](https://docs.gitcoin.co/mk_contributors/#docstrings)
- [ ] Completed: Code review by core team and any requested changes

###### Contributor

- [ ] Read and followed the [Contributor Guidelines](https://docs.gitcoin.co/mk_contributors/)
- [ ] Tested all changes **locally**
- [ ] Verified existing functionality
- [ ] Ran `make test` and everything passed!

###### Reviewer

- [ ] Affirm contributor guidelines have been followed and requested changes made
- [ ] CI tests and linting pass
- [ ] No conflicts (migrations, files, etc)
- [ ] Regression tested against staging

### CODE REVIEW

File: app/app/admin_urls.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""Handle admin specific URLs.
+
```
Comment (owocki) [inline app/app/admin_urls.py]: awesome :)

File: app/app/utils.py [hunk 2]
```diff
@@ -176,8 +176,7 @@ def sync_profile(handle, user=None, hide_profile=True):
     handle = handle.strip().replace('@', '')
     data = get_user(handle)
     email = ''
-    is_error = 'name' not in data.keys()
-    if is_error:
+    if 'name' not in data.keys():
```
Comment (owocki) [inline app/app/utils.py]: i personally prefer the specific `is_error` bc i think the naming convention tells you something about  the view... but i dont feel super super strongly about it

File: app/gas/urls.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+from django.urls import path, re_path
+
+from .views import (
+    gas, gas_calculator, gas_faq, gas_faucet_list, gas_guzzler_view, gas_heatmap, gas_history_view, gas_intro,
+)
+
```
Comment (owocki) [inline app/gas/urls.py]: awesome!

File: app/gas/views.py [hunk 4]
```diff
@@ -25,29 +26,22 @@
...
-lines = {
-    1: 'red',
-    5: 'orange',
-    60: 'green',
-    90: 'steelblue',
-    105: 'purple',
-    120: '#dddddd',
-    180: 'black',
-}
+lines = {1: 'red', 5: 'orange', 60: 'green', 90: 'steelblue', 105: 'purple', 120: '#dddddd', 180: 'black', }
```
Comment (owocki) [inline app/gas/views.py]: IMHO the old way is easier to read / mutate

Comment (mbeacom) [reply app/gas/views.py]: @owocki 
Gotcha - I hadn't realized we regularly changed these values.  I'll go ahead and switch it back.

Some considerations moving forward:

- we can split this variable off into a constants module and treat it as a module level variable
- capitalized naming convention
- all hex codes or defined color names for consistency
- validating modifications to this dict if it regularly changes via tests and/or dynamically store them somewhere (probably not the most favorable option)

File: app/app/admin_urls.py [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+    re_path(r'^viz/sunburst/(.*)?$', dataviz_d3_views.viz_sunburst, name='viz_sunburst'),
+    re_path(r'^viz/chord/(.*)?$', dataviz_d3_views.viz_chord, name='viz_chord'),
+    re_path(r'^viz/steamgraph/(.*)?$', dataviz_d3_views.viz_steamgraph, name='viz_steamgraph'),
+    re_path(r'^viz/circles/(.*)?$', dataviz_d3_views.viz_circles, name='viz_circles'),
+    re_path(r'^viz/sankey/(.*)?$', dataviz_d3_views.viz_sankey, name='viz_sankey'),
+    re_path(r'^viz/spiral/(.*)?$', dataviz_d3_views.viz_spiral, name='viz_spiral'),
+    re_path(r'^viz/heatmap/(.*)?$', dataviz_d3_views.viz_heatmap, name='viz_heatmap'),
+    re_path(r'^viz/calendar/(.*)?$', dataviz_d3_views.viz_calendar, name='viz_calendar'),
+    re_path(r'^viz/draggable/(.*)?$', dataviz_d3_views.viz_draggable, name='viz_draggable'),
+    re_path(r'^viz/scatterplot/(.*)?$', dataviz_d3_views.viz_scatterplot, name='viz_scatterplot'),
```
Comment (thelostone-mc) [inline app/app/admin_urls.py]: @mbeacom you sir have a fan

**Pull request closed without merging**